### DISCUSSION TITLE
Works locally, but after deploying, it is not displayed correctly

### DISCUSSION THREAD

**Discussion by maxdemirdilek** [alshedivat/al-folio#2533] [Q&A] [answered]
Locally my webpage works beautifully. However, after deploying it, [it looks](https://maxdemirdilek.github.io) as follows:
![Screenshot 2024-06-30 at 14 35 34](https://github.com/alshedivat/al-folio/assets/173602315/073b1aba-787c-4ec6-8dfb-1002a9282459).

What is wrong? How can I fix this issue?

I read the [FAQ page](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-webpage-works-locally-but-after-deploying-it-fails-to-build-and-throws-unknown-tag-toc-how-do-i-fix-that) but, as far as I can see, the `url` and `base url` paths are specified correctly.

As a side remark: It says "Some checks were not successful - (pages build and deployment / build (dynamic))" under "Actions". Maybe this error message indicates the cause of the problem ...

**Comment by george-gca** [comment, marked as answer]
Your urls in `_config.yml` seem to be fine, but looking at the `Actions` tab in your repo there are only `pages build and deployment`, which is weird. Looking at the console in your site, there are a lot of errors like this:

![image](https://github.com/alshedivat/al-folio/assets/31376482/5fdbfde3-3e24-4476-a8a8-4b69722d8cd4)

As you can see, it is trying to locate the assets under `https://maxdemirdilek.github.io/al-folio/`, which is wrong. This indicates that the latest uploaded version of your site was still with `baseurl` set as `/al-folio`.

[Our recommended approach](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#recommended-approach) is to first make it work as an exact copy of the repository, only them clone and customize it further. This way it is easier to pinpoint which commit introduced the issue. Since your site still have small modifications, I would recommend you save your site locally, start from scratch following strictly the [recommended approach](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#recommended-approach), then transfer your modifications to your new repo with the help of a tool like [meld](https://meldmerge.org/) or [winmerge](https://winmerge.org/).
